pages/storylets/harlowe/harlowe_storylets.md

```diff
@@ -0,0 +1,78 @@
+# "Storylets": Harlowe (v3.2)
+
+## Summary
+
+Storylets are [a different way of approaching nonlinear storytelling](https://emshort.blog/2019/11/29/storylets-you-want-them/). Instead of linking from one passage to another in Twine, storylets are defined by what are called its *requirements* and *content*.
+
+The metaphor of a deck of cards is often useful in understanding storylets. A card is able to be drawn from a deck if its requirements are met. Through defining many cards (passages in Twine) and conditions under which they can be used (their requirements), dynamic storytelling can take place as different arrangements become possible through writing simple rules for each card.
+
+The [`(storylet:)` macro](https://twine2.neocities.org/#macro_storylet) defines a passage as a storylet in Harlowe. It should appear as either the first line or as near to the top of the passage as possible. To define if it is available (called "open" in Harlowe) or not, it uses a [`when` lambda](https://twine2.neocities.org/#type_lambda) for determining, in the future, if the requirements are met using story and temporary variables.
+
+The [`(link-storylet:)` macro](https://twine2.neocities.org/#macro_link-storylet) selects the first open storylet based on a number passed to the macro. To access the fourth open storylet, for example, `(link-storylet: 4)` could be used. If no storylets are open, a third value can be used in the format of `(link-storylet: 4, "No storylets")` where the fourth open storylet would be used or the value "No storylets" shown to the user.
```

Comment:
Suggest changing the wording here as I feel it's a little confusing. Something like:
> The (link-storylet:) macro selects the first open storylet, or, if a number passed to the macro, the storylet associated with that number.

I think it needs clarified a bit is all.

Reply:
> Suggest changing the wording here as I feel it's a little confusing. Something like:
> 
> > The (link-storylet:) macro selects the first open storylet, or, if a number passed to the macro, the storylet associated with that number.
> 
> I think it needs clarified a bit is all.

I thought that was the case, too, but it does not work that way, it seems.

Using `(link-storylet:)` by itself produces the following error:

> the (link-storylet:) macro must only be given a whole number or a string or a "where ..." lambda, (optional) a whole number or a string or a "where ..." lambda and (optional) a string, in that order.